manifest/kcp.yaml

```diff
@@ -1,4 +1,17 @@
 ---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: kcp
+spec:
+  accessModes:
+  - ReadWriteOnce
```

Comment:
This is fine for now.

We might want to start thinking about a multi-shard deployment. Then we likely have to put the service account keys into a secret and share that secret between shards.

Reply:
We might also want to have service account key per shard, but distribute trust (some CA? Does that work with service accounts? cc @marun).

Reply:
https://github.com/kcp-dev/kcp/issues/799

Reply:
@sttts Token signing keys are rsa, so each server would either share the same key or have an independent key. There is no option for a certificate-based approach with CA afaik.